package.json

```diff
@@ -68,10 +68,10 @@
     "@babel/preset-env": "^7.16.4",
     "@babel/preset-react": "^7.16.0",
     "@storybook/addon-storysource": "^6.3.9",
```

Comment:
Looks like the latest version for `@storybook/addon-storysource` is v6.4.3

Reply:
ill fix that